build.sbt

```diff
@@ -0,0 +1,27 @@
+lazy val `sbt-circe-io` =
+  project.in(file(".")).enablePlugins(NoPublishPlugin).aggregate(core)
+
+// we're not doing code coverage on this plugin. Set to Some("sbt-circe-io") if you wish to run it.
+ThisBuild / circeRootOfCodeCoverage := None
+ThisBuild / tlCiScalafixCheck := false // we have no scalafix for the plugin
```

Comment:
In the interest of dog-fooding, you can enable it at the very least to keep the imports organized.

Reply:
Enabled